avaframe/com2AB/com2AB.py

```diff
@@ -245,6 +246,33 @@ def findSplitPoint(AvaProfile, splitPoint):
     return SplitPoint, splitPoint
 
 
+
+def readABinputs(cfgINPATH):
+    cfgAva = cfgINPATH['AvaLancheRoot']
+    cfgPath = {}
+
+    ProfileLayer = glob.glob(cfgAva + '/Inputs/LINES/*.shp')
+    cfgPath['ProfileLayer'] = ''.join(ProfileLayer)
+
+    DGMSource = glob.glob(cfgAva + '/Inputs/*.asc')
+    cfgPath['DGMSource'] = ''.join(DGMSource)

```

Comment:
Here we might have a potential Problem if more then 1 .asc files are present. 
We could either just take the first one, or throw an error if more than one .asc files are present (I tend more towards the second option...